wet.eos.basis.json

```diff
@@ -2,9 +2,75 @@
   "eft": "WET",
   "basis": "EOS",
   "metadata": {
-    "description": "Basis used by the EOS package. Neutrinos are in the flavour basis."
+    "description": "Basis used by the EOS software as of version 0.4 or later. Neutrinos are in the flavour basis."
   },
   "sectors": {
+    "sbsb": {
+      "sbsb::Re{c1}": {
+        "real": true,
+        "tex": "\\frac{4G_F}{\\sqrt{2}} (V_{tb} V_{ts}^*)^2 (\\bar s_L \\gamma^\\mu b_L)(\\bar s_L \\gamma_\\mu b_L)"
+      },
+      "sbsb::Im{c1}": {
+        "real": true,
+        "tex": "\\frac{4G_F}{\\sqrt{2}} (V_{tb} V_{ts}^*)^2 (\\bar s_L \\gamma^\\mu b_L)(\\bar s_L \\gamma_\\mu b_L)"
```

Comment:
If this is a real WC, there is an i (or rather -i) missing in the operator's tex (since it's defined as Lagrangian = -CO).

Reply:
The wilson coefficient is complex-valued in general. However, in EOS I use only real-valued parameters, which requires expressing this as two separate parameters in wcxf. This is identical to, e.g., the treatment of ``C_9`` in ``b->smumu``.
Neither my approach nor yours with the factor of ``i`` is wrong, I think. I declare one operator, and split the complex-valued WC into two-real valued parameters. You use two linearly dependent definitions of the operator, and keep only the real-part of their Wilson coefficients.

For what it's worth, there is precedent for my way, but I'm happy to convinced to use your way!

Reply:
You're right, my suggestion was wrong - I re-read the format definition and `real: true` is only admissible for Hermitian operators, so also using an i does not work. So the only way to conform with the standard is to use a complex WC with `Re` and `Im` keys. I understand that you only use real parameters, but what's the difference between having two separate keys and single key with `Re` and `Im` as subkeys (as required by the standard)?

Reply:
I had forgotten about the Re and Im part, and nobody caught this error until now. Have adapted the basis, and I will correct  ``wilson'' once this has been merged.

Reply:
Yes, it had slipped my attention as well.